_posts/2023-10-23-kubeflow-1.8-release.md

```diff
@@ -0,0 +1,374 @@
+
+
+### Platform dependencies, breaking changes, add-ons
+
+Kubeflow 1.8 includes hundreds of commits. The Kubeflow release process includes several rounds of testing by the Kubeflow working groups and Kubeflow distributions. Kubeflow’s configuration options provide a high degree of flexibility. After considering all of the testing options, the 1.8 Release Team narrowed the critical dependencies for consistent testing and documentation to the following.
+
+
+
+
+<table>
+  <tr>
+   <td><strong>Component</strong>
+   </td>
+   <td><a href="https://knative.dev/docs/reference/relnotes/">Knative</a>
+   </td>
+   <td><a href="https://istio.io/latest/news/releases/">Istio</a>
+   </td>
+   <td><a href="https://kubernetes.io/releases/">Kubernetes</a>
+   </td>
+   <td><a href="https://cert-manager.io/docs/installation/supported-releases/">Cert-Manager</a>
+   </td>
+   <td><a href="https://github.com/kubernetes-sigs/kustomize/releases">Kustomize</a>
+   </td>
+   <td><a href="https://github.com/dexidp/dex/releases">Dex</a>
+   </td>
+   <td><a href="https://github.com/argoproj/argo-workflows/releases">Argo</a>
+   </td>
+   <td>Tekton
+   </td>
+   <td>Oidc
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Version used in 1.8</strong>
+   </td>
+   <td>1.10.2 (Serving) 1.10.1 (Eventing)
+   </td>
+   <td>1.17.3
+   </td>
+   <td>1.25/1.26
+   </td>
+   <td>1.12.2
+   </td>
+   <td>5.0.3
+   </td>
+   <td>2.36.0
+   </td>
+   <td>3.3.10
+   </td>
+   <td>0.47.5
+   </td>
+   <td>e236439
+   </td>
+  </tr>
+</table>
+
+
+ \
+The 1.8 documentation includes overall installation instructions from the Manifest Working Group, and detailed feature reviews from each Kubeflow working group. Most of the working groups have broken their changelogs into subsections that highlight core features, UI enhancements, miscellaneous updates, bug fixes and breaking changes.
+
+
+## Updated Images, platform dependencies, breaking changes, and add-ons
+
+Kubeflow 1.8 includes hundreds of commits. The Kubeflow release process includes several rounds of testing by the Kubeflow working groups and Kubeflow distributions. Kubeflow’s configuration options provide a high degree of flexibility. After considering all of the testing options, the 1.8 Release Team narrowed the critical dependencies for consistent testing and documentation to the following.
+
+1.8 includes new notebook images with updates to support multi-architectures including ARM and Power processors, and updates to Tensorflow, PyTorch and other packages.
```

Comment:
This section can be removed, as it says nothing that the above sections don't already say.

Also, the first part is actually a direct copy from a few lines up.

```suggestion
```